pkg/util/round_precision_test.go

```diff
@@ -0,0 +1,57 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/stretchr/testify/assert"
+)
+
+func number(a interface{}) fixedpoint.Value {
+	switch v := a.(type) {
+	case string:
+		return fixedpoint.MustNewFromString(v)
+	case int:
+		return fixedpoint.NewFromInt(int64(v))
+	case int64:
+		return fixedpoint.NewFromInt(int64(v))
+	case float64:
+		return fixedpoint.NewFromFloat(v)
+	}
+
+	return fixedpoint.Zero
+}
+
+func Test_RoundPrice(t *testing.T) {
```

Comment:
name it as `Test_RoundAndTruncatePrice`

`Test_{func name}` is the convention

Reply:
Done.